frontend/src/common/types/ui/icon-name.type.ts

```diff
@@ -6,6 +6,11 @@ type IconName =
   | 'billing'
   | 'settings'
   | 'cross'
+  | 'nodejs'
+  | 'java'
+  | 'c'
+  | 'typescript'
+  | 'python'
```

Comment:
```suggestion
```
shouldn't be a part of the icon. should be in the `public` folder and use as static images